sdks/python/apache_beam/transforms/sql_test.py

```diff
@@ -209,6 +209,30 @@ def test_map(self):
           | SqlTransform("SELECT * FROM PCOLLECTION WHERE shopper = 'alice'"))
       assert_that(out, equal_to([('alice', {'apples': 2, 'bananas': 3})]))
 
+  def test_sql_ddl_set_option(self):
+    with TestPipeline() as p:
+      input_data = [
+          beam.Row(id=1, value=10),
+          beam.Row(id=2, value=20),
+          beam.Row(id=3, value=30)
+      ]
+      # DDL uses SET to modify a session option (tests DDL parsing)
+      # Using a known Calcite option like sqlConformance
+      ddl_statement = """
```

Comment:
this is more like a dumpy test. Not very familiar with the DDL support under Beam. Use this dumpy one for now.